features/step_definitions/assertion_steps.rb

```diff
@@ -0,0 +1,7 @@
+Then("I should see {string}") do |text|
+    expect(page).to have_content text
+end
+  
+Then("I should see {int}") do |number|
+    expect(page).to have_content number
+end
```

Comment:
Huh? Aren't those 2 definitions the same? Or am I missing something? I see that you use `string` in one of them and `int` in the other, but I fail to understand why?

Reply:
Please mind the code formatting/indentation

Reply:
What do you mean here? We use tab for indents.

Reply:
eh, i think it is something to do with settings in VS Code, we will research that. I have fixed the file now and defined that a tab is 2 spaces. Was under the impression that indentation setting should be the same for all files automatically, who knew.